editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/propertysheet/PropertiesUtils.java

```diff
@@ -311,7 +311,8 @@ public static void updateURIParams(Endpoint selectedEP, UriParameter p, Object v
     			}
     			if (val.trim().length()<1) val = pparam.getDefaultValue();
     			if (val != null && val.startsWith("/") && !CamelComponentUtils.isFileProperty(pparam)) val = val.substring(1);
-    			if (val != null) syntax = syntax.replace(pparam.getName(), val);
+    			// Workaround for FUSETOOLS-1369
+    			if (val != null) syntax = replace(syntax, pparam.getName(), val, syntax.indexOf(':'));
```

Comment:
I know that ':' is not always the delimeter but in all components listed in current Fuse palette it is used to differ protocol name from the rest of uri.
In your example 'host:port/path' only the part ':port/path' will change which is correct, we don't want to change the protocol name ('host').